Slim/Control/Queries.pm

```diff
@@ -5560,6 +5560,7 @@ my %tagMap = (
 	  'P' => ['genre_ids',         '',                'genres',        'id'],           #->genre_track->genres.id
 
 	  'k' => ['comment',           'COMMENT',         'comment'],                       #->comment_object
+	  '2' => 1,									    # to trigger addition of the input parameter
```

Comment:
That's not needed, is it? As it's always treated as a special case, `$colMap{2}` IMHO is never evaluated.